src/components/__tests__/Note.ts

```diff
@@ -1,110 +1,530 @@
-import { screen } from '@testing-library/dom'
+import { fireEvent, screen } from '@testing-library/dom'
 import { act } from 'react'
 import SimplePath from '../../@types/SimplePath'
 import { editThoughtActionCreator as editThought } from '../../actions/editThought'
 import { importTextActionCreator as importText } from '../../actions/importText'
+import { toggleNoteActionCreator as toggleNote } from '../../actions/toggleNote'
 import createTestApp, { cleanupTestApp } from '../../test-helpers/createTestApp'
 import dispatch from '../../test-helpers/dispatch'
 import { setCursorFirstMatchActionCreator as setCursor } from '../../test-helpers/setCursorFirstMatch'
 
 beforeEach(createTestApp)
 afterEach(cleanupTestApp)
 
-it('basic note', async () => {
-  await dispatch([
-    importText({
-      text: `
+describe('=note', () => {
+  test('basic', async () => {
+    await dispatch([
+      importText({
+        text: `
       - a
         - =note
           - foo`,
-    }),
-    // this will hide the meta attribute, so if the note value can be selected on the screen it must be rendered
-    setCursor(null),
-  ])
+      }),
+      // this will hide the meta attribute, so if the note value can be selected on the screen it must be rendered
+      setCursor(null),
+    ])
 
-  await act(vi.runOnlyPendingTimersAsync)
+    await act(vi.runOnlyPendingTimersAsync)
 
-  const element = screen.getByText('foo')
-  expect(element)
-})
+    // Verify note is rendered
+    const noteElement = screen.queryByLabelText('note')
+    expect(noteElement).toBeInTheDocument()
+
+    // Verify note content
+    const element = screen.getByText('foo')
+    expect(element)
+  })
 
-it('re-render note when =note subthought value changes', async () => {
-  await dispatch([
-    importText({
-      text: `
+  test('re-render note when =note subthought value changes', async () => {
```

Comment:
This test should be removed, per https://github.com/cybersemics/em/pull/2951#discussion_r2106224289.